main.py

```diff
@@ -0,0 +1,77 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
+import json
+import requests
+
+
+# Define the configuration for the Chatbot
+# Replace the placeholder values with your own email, password, and/or session token
+config = {
+  "session_token":
+  "eyJhb2fuV2KvwtQ"
+}
+
+class RequestHandler(BaseHTTPRequestHandler):
+    def do_POST(self):
+        # Get the length of the request body
+        content_length = int(self.headers['Content-Length'])
+        
+        # Read the request body
+        body = self.rfile.read(content_length)
+        
+        # Handle the webhook
+        self.handle_webhook(body)
+        
+        # Send a response
+        self.send_response(200)
+        self.end_headers()
+        self.wfile.write(b"Received POST request")
+
+
+    def handle_webhook(self, body):
+        # Decode the request body
+        data = body.decode('utf-8')
+
+
+        print("Received POST request")
+
+        # Parse the data as a JSON object
+        data = json.loads(data)  
+
+        # Extract the pull request information from the data
+        pull_request = data['pull_request']
+
+        # Create the modified files array
+        modified_files = self.create_modified_files_array(pull_request)
+
+        print(modified_files)
+
+    
+        return modified_files
+
+    def create_modified_files_array(self, pull_request):
```

Comment:
[main.py, line 51]: The `create_modified_files_array` function retrieves the full code of each modified file from the pull request, but does not save the file names. This means that the caller of this function will not be able to access the file names of the modified files.